Dealer/Dealer.js

```diff
@@ -0,0 +1,130 @@
+const version = '1.0.0';
```

Comment:
Hi Keith,
you should probably move this part into some inner scope (like an on('ready') callback). Variables in the outermost scope are shared between all API scripts, so no other script could define a "version" variable (they also shouldn't do that, but better to be defensive here) and the API would fail with a syntax error.

Another thing, it's a good idea to begin your script with a comment due to the way all scripts are concatenated.